src/controllers/api.controller.ts

```diff
@@ -285,17 +288,30 @@ export const searchNodes = async (
   next: NextFunction
 ) => {
   try {
+
+    /* TODO TODO TODO : why are we using a search when we hav ethe claim id ?????????? */
+    /* this function can exist but it should not be called when we already know the claim id */
+    /* it is currently being called from front end to see graph view of a known claim */
+    /* front end shoudl change to pull by id, we need a new route to retrieve by id  or specific subject */
+    
     const { search, page = 0, limit = 0 } = req.query;
+
+    let clean_search = ''
+    if (typeof search === 'string') {
+      // for some reason they are getting double-encoded 
+      clean_search = decodeURIComponent(search);
```

Comment:
@Kholoudxs55kh @sarahassan0 this is the fix we needed for the items with slashes including linkedin urls ^^